### DISCUSSION TITLE
초기 세팅 시 WebException 에러 문제

### DISCUSSION THREAD

**Discussion by njngwn** [naverz/zepeto-studio-kor#787] [5. Q&A]
![image](https://user-images.githubusercontent.com/48943487/169927140-2356cbf3-70d4-4993-bc59-5b6bb462dfbe.png)
 
안녕하세요. 월드 개발을 해보기 위해서 유니티를 설치하고, 가이드에 나와있는 제페토 유니티패키지 파일을 임포트하려는데 에러가 나네요.

Unity는 2020.3.9f1 버전으로 사용했고,
유니티 패키지 파일은 가이드 웹페이지에 있는 링크를 바로 다운받았습니다.

비슷한 문제를 겪은 분이 계시던데, 패키지 파일을 설치를 해야 업데이트할 수 있을 것 같아서 다시 질문 드립니다.
감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Internel Server Error의 경우 네트워크나 방화벽 이슈로 추정됩니다.
사내 보안망이시라면 현재로서는 개발이 힘들 것 같습니다.
다만 로드맵 상에 저희가 서버를 제공하는 쪽으로 구성할 계획이 있어서 추후에는 이용이 가능해질 예정입니다. 
적용 시기는 미정인 부분 양해 부탁 드립니다.
감사합니다.

**Reply by njngwn** [reply]
계속 다시해봐도 안됐었는데, 방금 다시 해보니까 되네요..! 일시적인 이슈였던 것 같습니다. 이후 비슷한 경험을 겪으실 분들이 계실지 몰라 덧글 남깁니다. 처음과 같은 상황이었는데, 지금은 되네요. 저와 같은 이슈를 겪으시게 되면 일정 시간 후에 재시도 해보시는 것을 추천드립니다.

**Reply by Zepeto-TechEvangelist** [reply]
확인 감사 드립니다~😄
